app/ide-desktop/lib/dashboard/src/authentication/service.ts

```diff
@@ -69,29 +69,29 @@ const AMPLIFY_CONFIGS = {
     userPoolWebClientId: auth.UserPoolWebClientId('1a7bner2d5ured09anh39sqhho'),
     domain: auth.OAuthDomain('npekin-enso-domain.auth.eu-west-1.amazoncognito.com'),
     ...BASE_AMPLIFY_CONFIG,
-  } satisfies Partial<auth.AmplifyConfig>,
+  },
   /** Configuration for @indiv0's Cognito user pool. */
   npekin2: {
     userPoolId: auth.UserPoolId('eu-west-1_FWbSTq3fV'),
     userPoolWebClientId: auth.UserPoolWebClientId('1e54ioss6ct6qfuek44g8hi1p0'),
     domain: auth.OAuthDomain('npekin2-enso-domain.auth.eu-west-1.amazoncognito.com'),
     ...BASE_AMPLIFY_CONFIG,
-  } satisfies Partial<auth.AmplifyConfig>,
+  },
   /** Configuration for @pbuchu's Cognito user pool. */
   pbuchu: {
     userPoolId: auth.UserPoolId('eu-west-1_jSF1RbgPK'),
     userPoolWebClientId: auth.UserPoolWebClientId('1bnib0jfon3aqc5g3lkia2infr'),
     domain: auth.OAuthDomain('pb-enso-domain.auth.eu-west-1.amazoncognito.com'),
     ...BASE_AMPLIFY_CONFIG,
-  } satisfies Partial<auth.AmplifyConfig>,
+  },
   /** Configuration for the production Cognito user pool. */
   production: {
     userPoolId: auth.UserPoolId('eu-west-1_9Kycu2SbD'),
     userPoolWebClientId: auth.UserPoolWebClientId('4j9bfs8e7415erf82l129v0qhe'),
     domain: auth.OAuthDomain('production-enso-domain.auth.eu-west-1.amazoncognito.com'),
     ...BASE_AMPLIFY_CONFIG,
-  } satisfies Partial<auth.AmplifyConfig>,
-}
+  },
+} satisfies Record<config.Environment, Partial<auth.AmplifyConfig>>
```

Comment:
This notation should provide advanced typesafety

Reply:
true... probably will be obsoleted by another PR soon though:
- https://github.com/enso-org/enso/pull/8895

Reply:
Ok, will omit the change then